js/src/dropdown.js

```diff
@@ -174,7 +175,12 @@ class Dropdown extends BaseComponent {
         }
       }
 
+      // The `.popper-active` class removes the `right` property in CSS
+      // This way popper can position the dropdown correctly
+      // We add it back after the calculations to get control over our positioning with our CSS again
+      this._menu.classList.add(CLASS_NAME_POPPER_ACTIVE)
       this._popper = Popper.createPopper(referenceElement, this._menu, this._getPopperConfig())
+      this._menu.classList.remove(CLASS_NAME_POPPER_ACTIVE)
```

Comment:
Why do we remove the class again? When the element if positioned by popper the `right` property will not have an effect anyway or am I wrong?

Reply:
That's what I would expect too, but it turns out this is needed by popper.